Sources/SwiftPlot/Renderer.swift

```diff
@@ -129,6 +129,23 @@ public protocol Renderer: AnyObject{
                          radius r: Float,
                          fillColor: Color)
 
+
+    /*drawSolidEllipse()
+    *params: center c: Point,
+    *        radius rx: Float,
+    *        radius rx: Float,
+    *        fillColor: Color,
+    *        isOriginShifted: Bool
+    *description: Draws an ellipse with specified fill color, center and radii
+    *             This function can operate in both coordinate systems with and
+    *             without shifted origin.
+    *             This is decided by the boolean parameter isOriginShifted.
+    */
+    func drawSolidEllipse(center c: Point,
+                          radius rx: Float,
+                          radius ry: Float,
```

Comment:
Could you change the name of the external labels for the radius? For someone using this function it will not be clear which parameter is the x or y radius unless they go look at this implementation.